syslog/modules/lpeg/sendmail.lua

```diff
@@ -0,0 +1,32 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Sendmail module
+
+## Variables
+### LPEG Grammars
+* `grammar`
+--]]
+
+local l = require "lpeg"
+l.locale(l) -- we want l.alpha etc...
+
+local rawset = rawset
+
+local M = {}
+setfenv(1, M) -- Remove external access to contain everything in the module
+
+local space = l.space^0
+local key = l.C(l.alpha^1) * space
+local value = l.C((l.P(1) - l.P', ')^1) * space
+-- l.R(" +","-~")^1) * space
+local sep = l.P',' * space
+local pair = l.Cg(key * "=" * value) * sep^-1
```

Comment:
inconsistent quote style

Reply:
fixed the quoting. It seems you're preferring double quotes.